src/views/ont/OntLibraries.vue

```diff
@@ -52,31 +62,36 @@ export default {
   },
   data () {
     return { 
-      fields: [
-        { key: 'id', label: 'ID' , sortable: true},
-        { key: 'name', label: 'Name', sortable: true},
-        { key: 'poolSize', label: 'Pool Size', sortable: true},
-        { key: 'tubeBarcode', label: 'Tube Barcode' , sortable: true},
-        { key: 'plateBarcode', label: 'Plate Barcode', sortable: true},
-        { key: 'pool', label: 'Pool #', sortable: true},
-        { key: 'createdAt', label: 'Created at', sortable: true},
-      ],
+      fields: [ 'id', 'name', 'poolSize', 'tubeBarcode', 'plateBarcode', 'pool', 'createdAt', 'assignedToFlowcell'],
       selected: [],
-      sortBy: 'createdAt',
-      sortDesc: true,
-    }
-  },
-  apollo: {
-    libraries: {
-      query: LIBRARIES_ALL_QUERY,
-      variables () {
-        return {
-          unassignedToFlowcells: true,
-        }
-      },
+      perPage: 5,
+      currentPage: 1,
+      totalRows: 0  
     }
   },
   methods: {
+    handlePrint(printer) {
+      this.handlePrintLabel(printer)
+    },
+    getLibraries(ctx, callback) {
+      this.$apollo.query({
+        query: LIBRARIES_ALL_QUERY,
+        variables: {
+          unassignedToFlowcells: false,
```

Comment:
Should this also be true? Unless I'm misunderstanding.

Reply:
Its getting all libraries, including a assignedToFlowcell field, which is being displayed in the front end table. This could be filtered on in the future